docs/en/reference/batch-processing.rst

```diff
@@ -183,7 +183,7 @@ problems using the following approach:
         // do stuff with the data in the row, $row[0] is always the object
     
         // detach from Doctrine, so that it can be Garbage-Collected immediately
-        $this->_em->detach($row[0]);
+        $this->_em->clear($row[0]);
```

Comment:
Passing arguments to clear is also deprecated 😄
We possibly need to check more things in this doc.

Reply:
Detaching an entity won't be possible afaik, so I guess we will remove those parts from the documentation.